configure.py

```diff
@@ -1483,14 +1483,15 @@ def get_clang_inline_threshold():
     print('Installed boost version too old.  Please update {}.'.format(pkgname("boost-devel")))
     sys.exit(1)
 
-if try_compile(args.cxx, source = textwrap.dedent('''\
+if not try_compile(args.cxx, source=textwrap.dedent('''\
         #include <lz4.h>
 
         void m() {
             LZ4_compress_default(static_cast<const char*>(0), static_cast<char*>(0), 0, 0);
         }
         '''), flags=args.user_cflags.split()):
-    defines.append("HAVE_LZ4_COMPRESS_DEFAULT")
+    print('Installed lz4-devel is too old. Please update it')
```

Comment:
Please also say what is the minimum version to update to here.

Reply:
@denesb hi Botond, thanks for your review. added and repushed.